<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>671</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>891</width>
      <height>238</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Клиенская база</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Фамилия</string>
        </property>
        <property name="font">
         <font>
          <family>Sitka Text</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Имя</string>
        </property>
        <property name="font">
         <font>
          <family>Sitka Text</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Номер </string>
        </property>
        <property name="font">
         <font>
          <family>Sitka Text</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Услуга</string>
        </property>
        <property name="font">
         <font>
          <family>Sitka Text</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата</string>
        </property>
        <property name="font">
         <font>
          <family>Sitka Text</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Время</string>
        </property>
        <property name="font">
         <font>
          <family>Sitka Text</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cтоимость (₽)</string>
        </property>
        <property name="font">
         <font>
          <family>Sitka Text</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Добавление клиента:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Фамилия:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_firstname"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Имя:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_thirdname"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Номер:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_number"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Услуга:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_lastservice">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Муж. стрижка </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Жен. стрижка</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Прическа</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Укладка</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Лечение волос</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Окрашивание</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Дата:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_data"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Время:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_time"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Стоимость:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_cost"/>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>Sitka Subheading</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Укажите параметры поиска:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Фамилия:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_firstname_2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Имя:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_thirdname_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Номер:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_number_2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Услуга:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Дата:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_data_2"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Sitka Subheading</family>
          </font>
         </property>
         <property name="text">
          <string>Время:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_time_2"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Sitka Text</family>
          </font>
         </property>
         <property name="text">
          <string>Стоимость:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_cost_2"/>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="font">
          <font>
           <family>Sitka Subheading</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_lastservice_2">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Муж. стрижка </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Жен. стрижка</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Прическа</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Укладка</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Лечение волос</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Окрашивание</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>126</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_edit_client">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
